tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -49,9 +49,36 @@ limitations under the License.
 #if GOOGLE_TENSORRT
 #include "tensorrt/include/NvInfer.h"
 
-//  Check if the types are equal. Cast to int first so that failure log message
-//  would work!
-#define CHECK_EQ_TYPE(val1, val2) CHECK_EQ((int)val1, (int)val2)
+// Check if the types are equal. Cast to int first so that failure log message
+// would work!
+#define TFTRT_CHECK_EQ_TYPE(val1, val2) CHECK_EQ((int)val1, (int)val2)
+
+#define TFTRT_RETURN_ERROR_IF_FALSE(ptr, node) \
+  do {                                         \
+    if (ptr == false) {                        \
```

Comment:
s/false/nullptr? Is ptr a pointer? If so what's the difference between this macro and the xx_IF_NULLPTR below? Otherwise please rename 'ptr' to something else.

Reply:
I'll rename ptr.